src/app/community/community.component.html

```diff
@@ -46,7 +46,12 @@ <h2 i18n>Communities List</h2>
     </ng-container>
     <ng-container matColumnDef="action">
       <mat-header-cell *matHeaderCellDef i18n > Actions </mat-header-cell>
-      <mat-cell *matCellDef="let element"><button md-raised-button (click)="deleteClick(element)">Delete</button></mat-cell>
+      <mat-cell *matCellDef="let element">
+        <button mat-raised-button (click)="updateClick(element, 'delete')">Delete</button>
```

Comment:
There is no translation in button. Could you please check it once ?
Eg; 
`<button mat-raised-button (click)="updateClick(element, 'delete')" i18n>Delete</button>`
use **i18n** for translation.